k8s/generate_code.sh

```diff
@@ -15,8 +15,12 @@ CRDS_DIR="crds"
 CONTROLLER_TOOLS_VERSION="v0.7.0"
 KUSTOMIZE_VERSION="kustomize/v3.10.0"
 
-if [[ -z "${GOPATH}" ]]; then
-  GOPATH=~/go
+if [[ -z "${GOPATH:-}" ]]; then
```

Comment:
Suggest moving `GOPATH` declaration to the variable declaration above, so you don't need to do this repeatedly in each condition expression. 

```shell
GOPATH="${GOPATH:-}"

if [[ -z "$GOPATH" ]]; then
  GOPATH="$(go env GOPATH)"
  
  if [[ -z "$GOPATH" ]]; then
    echo "GOPATH is not set"
    exit 1
  fi
fi
```

Reply:
Updated. Thank you.